modules/Sources/Features/Tabs/TabsStore.swift

```diff
@@ -144,13 +155,43 @@ public struct TabsReducer: Reducer {
                 state.isRestoringWallet = value
                 return .none
 
-            case .send(.sendDone):
-                state.selectedTab = .account
-                return .none
-            
+            case .send(.sendConfirmationRequired):
+                state.sendConfirmationState.amount = state.sendState.amount
+                state.sendConfirmationState.address = state.sendState.address
+                state.sendConfirmationState.proposal = state.sendState.proposal
+                state.sendConfirmationState.feeRequired = state.sendState.feeRequired
+                state.sendConfirmationState.message = state.sendState.message
+                return .send(.updateDestination(.sendConfirmation))
+                                
             case .send:
                 return .none
 
+            case .sendConfirmation(.sendPartial):
+                state.selectedTab = .send
+                return .none
+
+            case .sendConfirmation(.sendDone):
+                state.selectedTab = .account
+                return .merge(
+                    .send(.updateDestination(nil)),
+                    .send(.send(.resetForm))
+                    )
+                
+            case .sendConfirmation(.partialProposalError(.dismiss)):
+                //state.sendConfirmationState.partialProposalErrorViewBinding = false
+                //return .send(.updateDestination(nil))
```

Comment:
Is this commented-out part planned to be used? Or could we remove it?

Reply:
Good catch, removed